arcadia_pycolor/mpl.py

```diff
@@ -62,11 +64,45 @@ def _arcadia_fonts_found() -> bool:
     return len(arcadia_fonts) > 0
 
 
-def save_figure(fname: str, context: str = "web", **savefig_kwargs: dict[Any, Any]) -> None:
-    "Save the current figure with the default settings for web."
+def save_figure(
+    fname: str,
+    fname_types: Union[list[str], None] = None,
+    context: str = "web",
+    **savefig_kwargs: dict[Any, Any],
+) -> None:
+    """
+    Save the current figure, accounting for Arcadia's defaults.
+
+    Args:
+        fname (str): the filename to save the figure to
+        fname_types (list, optional): the filetypes(s) to save the figure to.
+            If None, the original suffix of fname is used.
+            If the original suffix is not in fname_types, it is appended to the list.
+        context (str): the context to save the figure in, either 'web' or 'print'
+        **savefig_kwargs: additional keyword arguments to pass to plt.savefig
+    """
     kwargs = SAVEFIG_KWARGS_WEB if context == "web" else SAVEFIG_KWARGS_PRINT
     kwargs.update(**savefig_kwargs)  # type: ignore
-    plt.savefig(fname=fname, **kwargs)  # type: ignore
+
+    # Gets a list of valid suffixes for saving figures from matplotlib.
+    valid_suffixes = list(FigureCanvasBase.get_supported_filetypes().keys())
```

Comment:
Nit: use `filetype` instead of `suffixes`
```suggestion
    valid_filetypes = list(FigureCanvasBase.get_supported_filetypes().keys())
```